linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/install_smartcard_packages/rule.yml

```diff
@@ -37,19 +44,21 @@ references:
     nist: CM-6(a)
     srg: SRG-OS-000105-GPOS-00052,SRG-OS-000375-GPOS-00160,SRG-OS-000375-GPOS-00161,SRG-OS-000377-GPOS-00162
     stigid@rhel7: RHEL-07-041001
+    stigid@sle12: SLES-12-030500
     stigid@rhel8: RHEL-08-010390
 
 ocil_clause: 'smartcard software is not installed'
 
-{{%- if product in ["rhel7", "ol7"] %}}
-ocil: '{{{ ocil_package(package="pam_pkcs11") }}}'
-{{%- else %}}
-ocil: '{{{ ocil_package(package="openssl-pkcs11") }}}'
-{{%- endif %}}
+ocil: |-
+    {{% for pkg in smartcard_packages %}}
+    '{{{ ocil_package(package=pkg) }}}'
+    {{% endfor %}}
 
+{{% if product != 'sle12' %}}
```

Comment:
If you would like to have a OVAL check and remediation exclusively to SLE12, it's possible to add files such as:
```
linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/install_smartcard_packages/oval/sle12.xml
linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/install_smartcard_packages/oval/bash/sle12.sh
linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/install_smartcard_packages/oval/ansible/sle12.yml
```

and the contents of it will be included in SLE12 product only.

Reply:
While currently OVAL check and remediation are exclusive to SLE12, within the move they will also apply to sle15,

Reply:
smart_card_login/install_smartcard_packages/rule.yml file has been removed from this PR.

Reply:
Ok